### PR TITLE
Merge branch-PPP-Benjamin with Master

### PR DESCRIPTION
**Pull request by Benjamin-Sim**
Updated PPP with project contributions

### CODE REVIEW

File: docs/team/benjamin-sim.md [hunk 1]
```diff
@@ -5,42 +5,60 @@ title: Benjamin Sim's Project Portfolio Page
...
-  * Updated the GUI color scheme (Pull requests [\#33](), [\#34]())
-  * Wrote additional tests for existing features to increase coverage from 88% to 92% (Pull requests [\#36](), [\#38]())
+    * Highlights: This feature required redesigning the original `list` command. Furthermore, designing how the sorting operation would function was challenging as it required detailed analysis of how data was handled and presented on the GUI.
+
+  * **New Feature**: Added the ability to sort events when listing them.
+    * What it does: Allows the user to sort all events in the application by their event title or date.
+
+    * Justification: Sorting events improves the user's productivity as he can easily locate events in the event list.
+
+    * Highlights: This featue required redesigning the `listEvents` command created by another team member.
```
Comment (Qiaoran-M) [inline docs/team/benjamin-sim.md]: Minor typo
```suggestion
    * Highlights: This feature required redesigning the `listEvents` command created by another team member.
```

File: docs/team/benjamin-sim.md [hunk 2]
```diff
@@ -5,42 +5,60 @@ title: Benjamin Sim's Project Portfolio Page
...
+
+
+  * **Testing**: Added substantial unit tests and integration tests for the sort persons feature, increasing test coverage from 48% to 50% (Pull request [#192](https://github.com/AY2223S1-CS2103-F09-2/tp/pull/192))
+
+  * **Testing**: Added some unit tests tests for the sort events feature. (Pull request [#192](https://github.com/AY2223S1-CS2103-F09-2/tp/pull/192))
```
Comment (Qiaoran-M) [inline docs/team/benjamin-sim.md]: Minor typo
```suggestion
  * **Testing**: Added some unit tests for the sort events feature. (Pull request [#192](https://github.com/AY2223S1-CS2103-F09-2/tp/pull/192))
```

File: docs/team/benjamin-sim.md [hunk 3]
```diff
@@ -5,42 +5,60 @@ title: Benjamin Sim's Project Portfolio Page
...
-  * Justification: This feature improves the product significantly because a user can make mistakes in commands and the app should provide a convenient way to rectify them.
-  * Highlights: This enhancement affects existing commands and commands to be added in future. It required an in-depth analysis of design alternatives. The implementation too was challenging as it required changes to existing commands.
-  * Credits: *{mention here if you reused any code/ideas from elsewhere or if a third-party library is heavily used in the feature so that a reader can make a more accurate judgement of how much effort went into the feature}*
+* **Code Contributed**: [RepoSense link](https://nus-cs2103-ay2223s1.github.io/tp-dashboard/?search=&sort=groupTitle&sortWithin=title&timeframe=commit&mergegroup=&groupSelect=groupByRepos&breakdown=true&checkedFileTypes=docs~functional-code~test-code~other&since=2022-09-16&tabOpen=true&tabType=authorship&tabAuthor=Benjamin-Sim&tabRepo=AY2223S1-CS2103-F09-2%2Ftp%5Bmaster%5D&authorshipIsMergeGroup=false&authorshipFileTypes=docs~functional-code~test-code~other&authorshipIsBinaryFileTypeChecked=false&authorshipIsIgnoredFilesChecked=false)
```
Comment (Qiaoran-M) [inline docs/team/benjamin-sim.md]: Maybe it's better to shorten the url here
```suggestion
* **Code Contributed**: [RepoSense link](https://nus-cs2103-ay2223s1.github.io/tp-dashboard/?search=benjamin-sim&breakdown=true)
```

File: docs/team/benjamin-sim.md [hunk 4]
```diff
@@ -5,42 +5,60 @@ title: Benjamin Sim's Project Portfolio Page
...
+<br><br>
 
-* **New Feature**: Added a history command that allows the user to navigate to previous commands using up/down keys.
+* **Enhancements Implemented**:
 
-* **Code contributed**: [RepoSense link]()
+  * **New Feature**: Added the ability to sort persons when listing them.
```
Comment (Qiaoran-M) [inline docs/team/benjamin-sim.md]: Pull request link can be added here
```suggestion
  * **New Feature**: Added the ability to sort persons when listing them (Pull requests [#53](https://github.com/AY2223S1-CS2103-F09-2/tp/pull/53), [#83](https://github.com/AY2223S1-CS2103-F09-2/tp/pull/83))
```

File: docs/team/benjamin-sim.md [hunk 5]
```diff
@@ -5,42 +5,60 @@ title: Benjamin Sim's Project Portfolio Page
...
-* **Project management**:
-  * Managed releases `v1.3` - `v1.5rc` (3 releases) on GitHub
+    * Justification: Sorting persons improves the user's productivity as he can easily locate people in the contact list.
 
-* **Enhancements to existing features**:
-  * Updated the GUI color scheme (Pull requests [\#33](), [\#34]())
-  * Wrote additional tests for existing features to increase coverage from 88% to 92% (Pull requests [\#36](), [\#38]())
+    * Highlights: This feature required redesigning the original `list` command. Furthermore, designing how the sorting operation would function was challenging as it required detailed analysis of how data was handled and presented on the GUI.
+
+  * **New Feature**: Added the ability to sort events when listing them.
```
Comment (Qiaoran-M) [inline docs/team/benjamin-sim.md]: PR link can be added here
```suggestion
  * **New Feature**: Added the ability to sort events when listing them (Pull request [#113](https://github.com/AY2223S1-CS2103-F09-2/tp/pull/113))
```

**Review by Qiaoran-M — CHANGES_REQUESTED** [review]
LGTM except for minor changes!

File: docs/team/benjamin-sim.md (hunk 3 above)
Comment (Benjamin-Sim) [reply docs/team/benjamin-sim.md]: I was also wondering how to shorten the URL. 😄 Great that you found out how to do it!

**Review by Qiaoran-M — APPROVED** [review]
LGTM!

**Pull request merged**